pipelines/batch/src/main/java/com/google/fhir/analytics/FhirEtl.java

```diff
@@ -70,6 +78,16 @@ public class FhirEtl {
    */
   private static final Map<String, String> tableDateColumn;
 
+  /**
+   * NumShards is an option in the merger pipelines to determine the number of output shards for
+   * each FHIR Resource Type. Here it's used in the ViewDefinition-to-Parquet functionality
+   */
+  private static final int NUM_SHARDS = 20;
+
+  private static ViewManager VIEW_MANAGER;
```

Comment:
nit: a non-final element should not be ALL_CAPS ([style guide](https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names)).

Reply:
Done.